test/sqllogictest/mzcompose.yml

```diff
@@ -8,19 +8,34 @@
 # by the Apache License, Version 2.0.
 
 version: '3.7'
+
+mzworkflows:
+  sqllogictest-fast:
+    steps:
+      - step: start-services
+        services: [postgres]
+      - step: wait-for-postgres
+        dbname: postgres
+      - step: run
+        service: sqllogictest-svc
+
 services:
-  sqllogictest:
+  sqllogictest-svc:
     mzbuild: sqllogictest
     volumes:
     - ../../:/workdir
-    propagate-uid-gid: true
```

Comment:
Losing this can cause files to get created as root in the workdir, which causes all sorts of headaches. Does something go wrong if you leave it in?

Reply:
I was getting a "unknown UID" error but that was before the container got an entrypoint (that is, when it was simply dropping you in a shell).

I have restored that line since with the entrypoint present it all runs as expected even with it.